<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UTFDialog</class>
 <widget class="QDialog" name="UTFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UTF Conversion Fixes</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,1,1">
    <item row="0" column="0">
     <widget class="QPushButton" name="latinButton">
      <property name="statusTip">
       <string>Import ISO Latin (extended ASCII) text</string>
      </property>
      <property name="text">
       <string>&amp;ISO Latin</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/images/document-import.png</normaloff>:/images/document-import.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+I</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="visibleCtlButton">
      <property name="statusTip">
       <string>Show visual control codes which have no visual effect</string>
      </property>
      <property name="text">
       <string>Show &amp;CTRL</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/images/convert-format.png</normaloff>:/images/convert-format.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+C</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="escAnsiButton">
      <property name="statusTip">
       <string>Convert all control codes to recommended encoding</string>
      </property>
      <property name="text">
       <string>&amp;ESC ANSI</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/images/convert-format.png</normaloff>:/images/convert-format.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+E</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="markErrorsButton">
      <property name="statusTip">
       <string>Mark all encoding error to see them</string>
      </property>
      <property name="text">
       <string>&amp;Mark Errors</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/images/convert-bad.png</normaloff>:/images/convert-bad.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+M</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="markWarnButton">
      <property name="statusTip">
       <string>Mark all encoding warnings</string>
      </property>
      <property name="text">
       <string>Mark &amp;Warn</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/images/convert-bad.png</normaloff>:/images/convert-bad.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+W</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="statusTip">
       <string>Close the UTF fix dialog</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UTFDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
